scripts/build/windows.bat

```diff
@@ -226,6 +233,10 @@ signtool verify /pa /v %package_output%\Etcher.exe
 
 upx -9 %package_output%\*.dll
 
+cd %package_output%^
+ && 7z a -tzip ..\installers\%package_name%.zip *^
```

Comment:
I see, I didn't know Windows provided any built-in zip support we could use programmatically, so I just headed straight to 7z. I'll give the PowerShell and VBS suggestions a try and will see how that goes.